### PR TITLE
Fix capitalization and punctuation

### LINKED ISSUE #1505: Correct inconsistent capitalization in Tempo documentation
Review the Tempo documentation to match sentence case and correct minor punctuation errors. 

Goals: 
* Make sure pages are consistent in capitalization use
* Correct punctuation

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:

Fixes punctuation and capitalization inconsistencies

**Which issue(s) this PR fixes**:
Fixes #1505 

**Checklist**
- [ ] Tests updated
- [ X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by zalegrala — COMMENTED** [review]
Looks reasonable to me.

File: docs/tempo/website/getting-started/example-demo-app.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ There are monolithic mode and microservices examples.
 The Helm [example](https://github.com/grafana/tempo/tree/main/example/helm) shows a complete microservice based deployment. 
 There are monolithic mode and microservices examples.
 
-## The New Stack (TNS) Demo
+## The New Stack (TNS) demo
 
 The [TNS demo](https://github.com/grafana/tns) demonstrates a fully-instrumented three-tier application and the integration of Grafana, Prometheus, Loki, and Tempo [features](https://github.com/grafana/tns#demoable-things), including metrics to traces (exemplars), logs to traces, and traces to logs.  A good place to start is the [docker-compose setup](https://github.com/grafana/tns/tree/main/production/docker-compose) which includes a pre-built dashboard, load generator, and exemplars.
```
Comment (knylander-grafana) [inline docs/tempo/website/getting-started/example-demo-app.md:39]:
```suggestion
## The New Stack demo

The [New Stack (TNS) demo](https://github.com/grafana/tns) demonstrates a fully-instrumented three-tier application and the integration of Grafana, Prometheus, Loki, and Tempo [features](https://github.com/grafana/tns#demoable-things), including metrics to traces (exemplars), logs to traces, and traces to logs. A good place to start is the [docker-compose setup](https://github.com/grafana/tns/tree/main/production/docker-compose) which includes a pre-built dashboard, load generator, and exemplars.
```

File: docs/tempo/website/api_docs/pushing-spans-with-http.md [hunk 2]
```diff
@@ -76,9 +76,9 @@ And open `http://localhost:16686/trace/0123456789abcdef` in the browser of your
 
 <p align="center"><img src="../pushing-spans-with-http.png" alt="single span"></p>
 
-## More Spans!
+## More spans!
```
Comment (KMiller-Grafana) [inline docs/tempo/website/api_docs/pushing-spans-with-http.md:79]: I advise against using exclamation points in all technical documentation.

File: docs/tempo/website/configuration/compression.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 weight: 5
 ---
 
-# Compression/Encoding
+# Compression and encoding
 
-Tempo has the ability to compress traces that it pushes into the backend. This requires a bit extra
-memory and cpu but seriously reduces the amount of stored data.  Anecdotal tests suggest that zstd will
+Tempo can compress traces that it pushes into the backend. This requires a bit extra
+memory and CPU but reduces the amount of stored data.  Anecdotal tests suggest that `zstd` will
```
Comment (KMiller-Grafana) [inline docs/tempo/website/configuration/compression.md:9]: I feel this wording is unpolished.  Here's a suggestion.  Not a necessary change.
```suggestion
Tempo can compress traces that it pushes to backend storage. This requires extra
memory and CPU, but it reduces the quantity of stored data.  Anecdotal tests suggest that `zstd` will
```

File: docs/tempo/website/configuration/manifest.md [hunk 4]
```diff
@@ -5,16 +5,16 @@ weight: 7
 
 # Manifest
 
-This doc contains all Tempo options and their defaults. It is likely quite overwhelming and if you are just getting 
-started it is recommended to check [Tempo examples](https://github.com/grafana/tempo/tree/main/example/docker-compose) 
+This document contains all Tempo options and their defaults. It is likely quite overwhelming and if you are just getting 
+started. Refer to [Tempo examples](https://github.com/grafana/tempo/tree/main/example/docker-compose)
```
Comment (KMiller-Grafana) [inline docs/tempo/website/configuration/manifest.md:9]:
```suggestion
This document contains all Tempo options and their defaults. It is likely quite overwhelming if you are just getting 
started. Refer to [Tempo examples](https://github.com/grafana/tempo/tree/main/example/docker-compose) 
```

File: docs/tempo/website/configuration/multitenancy.md [hunk 5]
```diff
@@ -48,8 +48,8 @@ exporters:
 - Multitenancy on ingestion is currently [only working](https://github.com/grafana/tempo/issues/495) with GPRC and this may never change. It is strongly recommended to use the OpenTelemetry Collector to support multitenancy as described above.
 - The way the read path is configured is temporary and should be much more straightforward once the [tempo-query dependency is removed](https://github.com/grafana/tempo/issues/382).
 
-## Enabling Multitenancy
-To enable multitenancy on Tempo backend, simply set the following config value on all Tempo components:
+## Enabling multitenancy
+To enable multitenancy on Tempo backend, set the following configuration value on all Tempo components:
```
Comment (KMiller-Grafana) [inline docs/tempo/website/configuration/multitenancy.md:52]: At Grafana Labs, we use the spelling  "multi-tenancy."  Please change this throughout.
```suggestion
## Enabling multi-tenancy
To enable multi-tenancy on Tempo backend, set the following configuration value on all Tempo components:
```

File: docs/tempo/website/operations/_index.md [hunk 6]
```diff
@@ -1,8 +1,10 @@
 ---
-title: Deployment and Operations
+title: Deployment and operations
 weight: 300
 ---
 
+# Deployement and operations
```
Comment (KMiller-Grafana) [inline docs/tempo/website/operations/_index.md:6]:
```suggestion
# Deployment and operations
```

**Review by KMiller-Grafana — APPROVED** [review]
There are a few suggestions here for improvement.  I approve with or without.

File: docs/tempo/website/api_docs/pushing-spans-with-http.md (hunk 2 above)
Comment (knylander-grafana) [inline docs/tempo/website/api_docs/pushing-spans-with-http.md:79]:
```suggestion
## More spans
```

File: docs/tempo/website/operations/consistent_hash_ring.md [hunk 7]
```diff
@@ -7,65 +7,63 @@ weight: 2
...
 **Path:** `/distributor/ring`
 
 Unhealthy distributors have little impact but should be forgotten to reduce cost of maintaining the ring .
 
-#### Ingester
+### Ingester
 **Available on:** Distributors  
 **Path:** `/ingester/ring`
 
 Unhealthy ingesters will cause writes to fail. If the ingester is really gone forget immediately!
```
Comment (knylander-grafana) [inline docs/tempo/website/operations/consistent_hash_ring.md:53]:
```suggestion
Unhealthy ingesters will cause writes to fail. If the ingester is really gone, forget it immediately.
```

File: docs/tempo/website/operations/consistent_hash_ring.md [hunk 8]
```diff
@@ -7,65 +7,63 @@ weight: 2
...
 **Path:** `/metrics-generator/ring`
 
 Unhealthy metrics-generators will cause writes to fail. If the metrics-generator is really gone, forget it immediately.
 
-#### Compactor
+### Compactor
 **Available on:** Compactors  
 **Path:** `/compactor/ring`
 
 Unhealthy compactors will allow the blocklist to grow significantly. If the compactor is really gone forget immediately!
```
Comment (knylander-grafana) [inline docs/tempo/website/operations/consistent_hash_ring.md:65]:
```suggestion
Unhealthy compactors will allow the blocklist to grow significantly. If the compactor is really gone, forget it immediately.
```

**Pull request merged**